mne/gui/_coreg.py

```diff
@@ -1016,7 +1025,8 @@ def _set_sensors_visibility(self, state):
 
     def _update_actor(self, actor_name, actor):
         # XXX: internal plotter/renderer should not be exposed
-        self._renderer.plotter.remove_actor(self._actors.get(actor_name))
+        self._renderer.plotter.remove_actor(self._actors.get(actor_name),
+                                            render=False)
```

Comment:
FYI this made me realize we had a lot of these elsewhere. This gets rid of the bug where things "flash" a bit during updates. Now they just update as I'd expect!

PR to fix other instances is in https://github.com/mne-tools/mne-python/pull/10423

Reply:
> This gets rid of the bug where things "flash" a bit during updates.

I did not notice those but I'm glad it's taken care of